src/cdk-experimental/selection/select-all.ts

```diff
@@ -60,55 +64,43 @@ export class CdkSelectAll<T> implements OnDestroy, OnInit {
     });
   }
 
-  private readonly _destroyed$ = new Subject();
+  private readonly _destroyed = new ReplaySubject(1);
```

Comment:
Why make this a `ReplaySubject`?

Reply:
Our internal Angular style guide actually calls this out under the "Patterns for unsubscribing" section.
>> ReplaySubject also handles cases where a subscription takes place after the component is already destroyed
It is not very important. I will change it back to `Subject` since it seems to be the convention here.